Sources/Hammer/AppleInternal/AppleInternal+IOHID.swift

```diff
@@ -46,8 +44,10 @@ struct IOHID {
 
     typealias IOHIDEventSystemClientCreate = @convention(c) (
         _ allocator: CFAllocator?) -> IOHIDEventSystemClient
+    // swiftlint:disable:next type_name
     typealias IOHIDEventSystemClientScheduleWithRunLoop = @convention(c) (
         _ client: IOHIDEventSystemClient, _ runloop: CFRunLoop, _ mode: CFRunLoopMode.RawValue) -> Void
+    // swiftlint:disable:next type_name
```

Comment:
Lets undo these changes. This file is a collection of external mappings so we can disable the linter for the whole file safely

Reply:
Done

Reply:
[The last time](https://github.com/lyft/Hammer/actions/runs/4833174251/jobs/8613101532) `swiftlint` emitted an error with no file path. I cannot reproduce that locally (swiftlint 0.51.0 – the latest):

```
Error: Prefer For-Where Violation: `where` clauses are preferred over a single `if` inside a `for` (for_where)

```